### DISCUSSION TITLE
Support for multiple interfaces? (e.g. external 2.4 & int 5.8ghz)

### DISCUSSION THREAD

**Discussion by smanley** [RaspAP/raspap-webgui#933] [Q&A]
Hi;

Is it possible to use an external 2.4ghz USB adapter with the internal wifi (e.g. set to 5.8ghz) to run both banks from one Pi 4?

e.g. one SSID on 2.4ghz USB and another SSID on internal 5.8 wifi adapter.

I didn't seem to see a way to do this from the GUI, but perhaps I'm missing something.

Thanks,
Steve

**Comment by billz** [comment]
Hi Steve,

Great question. This discussion offers a starting point https://github.com/RaspAP/raspap-webgui/discussions/760#discussioncomment-285110. `hostapd` in used in interactive mode in this example. This manual setup works well with a second adapter (an Edimax EW-7811Un is demonstrated here). 

<img width="489" alt="dual-bands" src="https://user-images.githubusercontent.com/229399/121822121-a380ef80-cc9d-11eb-94a1-d404adc07b78.png">

There is no GUI for this at the moment, but if there's enough interest it could be added to the project roadmap.

Hope this is useful.

**Reply by smanley** [reply]
Cool.

I think I was the first subscribing sponsor - maybe this feature request could be poked into the list? 

I can see this being a very common use case.

I’ll have a look at the manual config. 

Thanks for the speedy reply.

Steve

**Reply by billz** [reply]
> maybe this feature request could be poked into the list?

Done. I've created an issue in the Insiders repo to track this https://github.com/RaspAP/raspap-insiders/issues/38

In the meantime, I think it would be helpful to add a tutorial to the docs.

> I think I was the first subscribing sponsor

You were indeed :) Thank you for sponsorship!

**Reply by billz** [reply]
For clarity https://docs.raspap.com/multiple.html

**Reply by smanley** [reply]
Awesome, thanks.
